src/timeDiff.md

```diff
@@ -0,0 +1,53 @@
+# [Snippet Name]
+Returns the time difference between input and current time.
+
+## Result
+```js
+  timeDiff("2022-10-27 17:30:00"); // 1 hour 5 minutes
+  // Supposing that the current time is 18:35
+```
+
+## Snip Langauge
+* [JavaScript](#javascript)
+
+### JavaScript
+```js
+  const timeDiff = (input) => {
+    const [date, time] = input.split(' ')
+    const [year, month, day] = date.split('-')
+    const [hour, minute, second] = time.split(':')
```

Comment:
Add default values for time (for seconds at least)

💡In case the user does not pass the value